analyze/pubspec.yaml

```diff
@@ -11,5 +11,5 @@ dependencies:
   platform: 3.1.6
 
 dev_dependencies:
-  mockito: 5.4.4
```

Comment:
Looks like this one might need to stay at 5.4.4 for... old build package or something? https://ci.chromium.org/ui/p/flutter/builders/try/Linux%20Cocoon/6890/overview

Reply:
Let me try bumping!